### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by jameswonlee**
Basic functionality of navbar including logo (home), hamburger menu (open and close), navigation in hamburger menu options, and shopping cart (open and close).

### CODE REVIEW

File: components/NavBar/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    }, [showToolBox]);
+
+    useEffect(() => {
+        if (!showMenu) return;
+        document.addEventListener('click', closeMenu);
+        return () => document.removeEventListener("click", closeMenu);
+    }, [showMenu]);
+
+    return (
+        <div className={s.navbarContainer}>
```
Comment (SeanPaulson) [inline components/NavBar/navbar.tsx:59]: Looks like you will need to change display to sticky on this container.
Go to profile page and try it with fixed and sticky and let me know what you think.

**Comment by SeanPaulson** [comment]
Looks like the container div in the Navbar.tsx component does need to be position sticky.
Try going to the /profile page and try it with sticky and fixed and let me know what you think.

**Comment by jameswonlee** [comment]
Oh yeah. It looks like sticky works better than fixed. I made the changes and created a new PR. Thanks!

**Review by SeanPaulson — COMMENTED** [review]
Looks good!

**Comment by jameswonlee** [comment]
Thanks! I’m not sure if you saw but I closed PR #45. I created a new PR # 47. I wasn’t sure if that was the correct way of doing things, or if I should have just left PR #45 open with the changes I made to the CSS positioning.

**Pull request closed without merging**